scilpy/tracking/tools.py

```diff
@@ -36,23 +33,23 @@ def filter_streamlines_by_length(streamlines,
         List of filtered streamlines by length.
 
     filtered_data_per_point: dict
-        dict of data per point for filtered streamlines
+        dict of data per point for filtered streamlines.
     filtered_data_per_streamline: dict
-        dict of data per streamline for filtered streamlines
+        dict of data per streamline for filtered streamlines.
     """
 
     lengths = []
-    for streamline in streamlines:
+    for streamline in sft.streamlines:
         lengths.append(length(streamline))
 
     lengths = np.asarray(lengths)
 
     filter_stream = np.logical_and(lengths >= min_length,
                                    lengths <= max_length)
 
-    filtered_streamlines = list(np.asarray(streamlines)[filter_stream])
-    filtered_data_per_point = data_per_point[filter_stream]
-    filtered_data_per_streamline = data_per_streamline[filter_stream]
+    filtered_streamlines = list(np.asarray(sft.streamlines)[filter_stream])
```

Comment:
Casting streamlines as a numpy array ? This will crash if streamline do not have the same number of point, right ?